apps/bfd-server/bfd-server-docs/src/main/java/gov/cms/bfd/datadictionary/CsvToExcelCommand.java

```diff
@@ -0,0 +1,326 @@
+      new int[] {
+        2080, 7200, 10400, 2400, 1600, 1600, 5600, 11200, 14400, 17600, 17600, 17600, 9600, 9600,
+        6400, 6400, 6400, 6400, 6400, 2400, 2400, 2400, 2400, 2400, 2400
+      };
+
+  /**
+   * Creates, formats and saves an Excel workbook (xlsx) file.
+   *
+   * @param v1CsvPathName pathname of the V1 data dictionary CSV file
+   * @param v2CsvPathName pathname of the V2 data dictionary CSV file
+   * @param xslPathName pathname of the Excel file to output
+   */
+  public static void createAndSaveWorkbook(
+      String v1CsvPathName, String v2CsvPathName, String xslPathName) {
+    try (OutputStream os = new FileOutputStream(xslPathName);
+        Workbook wb = new XSSFWorkbook()) {
+
+      // create and format V1 sheet
+      var v1Sheet = wb.createSheet("V1");
+      importSheet(v1CsvPathName, v1Sheet);
+      formatSheet(wb, v1Sheet, v1ColumnsWidths);
+
+      // create and format V2 sheet
+      var v2Sheet = wb.createSheet("V2");
+      importSheet(v2CsvPathName, v2Sheet);
+      formatSheet(wb, v2Sheet, v2ColumnsWidths);
+
+      // save Excel file
+      wb.write(os);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  /**
+   * Applies formatting to data dictionary workbook sheet.
+   *
+   * @param wb the data dictionary Excel workbook
+   * @param sheet the data dictionary sheet to format
+   * @param columnsWidths array of column widths to apply to sheet
+   */
+  private static void formatSheet(Workbook wb, Sheet sheet, int[] columnsWidths) {
+    var maxRowIndex = sheet.getLastRowNum();
+    var maxColIndex = columnsWidths.length - 1;
+    searchAndReplace(sheet, CellRangeAddress.valueOf("D2:D" + (maxRowIndex + 1)), ";", "\n");
+    searchAndReplace(sheet, CellRangeAddress.valueOf("I2:J" + (maxRowIndex + 1)), ";", "\n");
+    setStyle(sheet, new CellRangeAddress(0, 0, 0, maxColIndex), getHeaderCellStyle(wb));
+    setStyle(sheet, new CellRangeAddress(1, maxRowIndex, 0, maxColIndex), getDataCellStyle(wb));
+    rowShading(sheet, new CellRangeAddress(1, maxRowIndex, 0, maxColIndex), getRowShadingStyle(wb));
+    setColumnWidths(sheet, columnsWidths);
+    setRowHeights(sheet, new CellRangeAddress(0, maxRowIndex, 0, maxColIndex));
+    sheet.createFreezePane(2, 1);
+  }
+
+  /**
+   * Load an CSV file into a workbook sheet.
+   *
+   * @param csvFileName the CSV file to import
+   * @param sheet the workbook sheet to receive the imported data
+   */
+  private static void importSheet(String csvFileName, Sheet sheet) {
+    var rowIndex = 0;
+    try (BufferedReader reader = new BufferedReader(new FileReader(csvFileName))) {
+      Iterable<CSVRecord> records = CSVFormat.EXCEL.parse(reader);
+      for (CSVRecord record : records) {
+        Row row = sheet.createRow(rowIndex);
+        for (int i = 0; i < record.size(); i++) {
+          Cell cell = row.createCell(i);
+          cell.setCellValue(record.get(i));
+        }
+        rowIndex += 1;
```

Comment:
Tomato-tomato....(rowIndex++) ; could just use increment operator since it's being used in the for loop for columns.